### DISCUSSION TITLE
Request parameters don't work

### DISCUSSION THREAD

**Discussion by gjm** [dedoc/scramble#476] [Q&A]
If I understand correctly reading the documentation, parameters that are manually read from a request object should be documented. I have an invocable controller with something like this:

```php
public function __invoke(Request $request)
{
    $filter = $request->get('filter');
    ...
}
```

The `filter` parameter should be added to the endpoint documentation, right? Well, it doesn't. 

Maybe I'm missing something?

Thanks in advance.

**Comment by romalytvynenko** [comment]
@gjm What is the version of Scramble you're using?

**Reply by gjm** [reply]
@romalytvynenko I *was* using 0.8.6. 

It was installing an older version due to a dependency (`nikic/php-parser`) version conflict with package `barryvdh/laravel-ide-helper`. I was now able to install 0.11.

But now I'm getting an event stranger error...

While analysing some class that has traits it seems to "think" the use import of the trait is duplicated...

`Cannot use Authenticatable as Authenticatable because the name is already in use on line 17`

**Reply by gjm** [reply]
Tracked down the issue and created a PR ( #477 ) to address it.

**Reply by romalytvynenko** [reply]
@gjm thank you! This is awesome!
